docs/content/doc/advanced/adding-legal-pages.en-us.md

```diff
@@ -0,0 +1,38 @@
+---
+date: "2019-12-28"
+title: "Adding Legal Pages"
+slug: adding-legal-pages
+weight: 9
+toc: false
+draft: false
+menu:
+  sidebar:
+    parent: "advanced"
+    name: "Adding Legal Pages"
+    identifier: "adding-legal-pages"
+    weight: 9
+---
+
+Some jurisdictions (such as EU), requires certain legal pages (e.g. Privacy Policy) to be added to website. Follow these steps to add them to your Gitea instance.
+
+## Getting Pages
+
+Gitea source code ships with sample pages, available in `contrib/legal` directory. Copy them to `custom/public/`. For example, to add Privacy Policy:
+
+```
+cp $GOPATH/src/code.gitea.io/gitea/contrib/legal/privacy.html.sample /path/to/custom/public/privacy.html
```

Comment:
i would replace this with the online link

Reply:
@6543 did you mean replacing this `cp` line with following?:
```suggestion
wget -O /path/to/custom/public/privacy.html https://github.com/go-gitea/gitea/blob/master/contrib/legal/privacy.html.sample
```